ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/wms_detail.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Map configuration via Web Map Service(WMS)</title>
+</head>
+<body>
+	<h1>WMS Configuration</h1>
+	<p>
+		The <i>WMS</i> provides functionality to add map using service URL. 
+	</p>
+	<p>
+		Here is the explanation of how to add, configure and remove <i>WMS</i> 
+		map in <i>Hale</i> studio. To work with WMS maps, first open Map perspective 
+		using Open Perspective tool menu reside in the right corner of the studio.
+    </p>
+    
+    <div>
+		<img src="../images/map_perspective_toolbar.png"
+			title="Map perspective tool menu" class="fit">
+		<img src="../images/map_perspective_open.png"
+			title="Map perspective tool menu" class="fit" style="margin-left:20px" >
+	</div>
+    
+    <p>
+    	To add the WMS maps in to Map perspective, click on the <i>View Menu</i> 
```

Comment:
replace sentence with:

To add a WMS to the Map view, click on the <i>View Menu</i> and then pick <i>WMS maps</i>&#8594;<i>Add</i>